client/components/data/query-site-stats/index.jsx

```diff
@@ -38,7 +38,7 @@ class QuerySiteStats extends Component {
 
 		this.props.requestSiteStats( siteId, statType, query );
 		this.clearInterval();
-		if ( heartbeat, isAutoRefreshAllowedForQuery( query ) ) {
+		if ( heartbeat && isAutoRefreshAllowedForQuery( query ) ) {
```

Comment:
This almost looks too intentional to be an accident. @youknowriad did you intentionally use the comma operator here?